static/app/components/searchQueryBuilder/tokens/freeText.tsx

```diff
@@ -134,29 +134,7 @@ function replaceFocusedWordWithFilter(
   return value;
 }
 
-/**
- * Takes a string that contains a filter value `<key>:` and replaces with any aliases that may exist.
- *
- * Example:
- * replaceAliasedFilterKeys('foo issue: bar', {'status': 'is'}) => 'foo is: bar'
- */
-function replaceAliasedFilterKeys(value: string, aliasToKeyMap: Record<string, string>) {
```

Comment:
Instead of modifying this function to support the new getFieldDefinition, I deleted it because we are no longer using aliases.